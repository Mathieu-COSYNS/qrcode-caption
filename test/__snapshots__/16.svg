<svg xmlns="http://www.w3.org/2000/svg" width="500" height="575.8620689655172" viewBox="0 0 29 33.4" shape-rendering="crispEdges"><rect fill="#ff0000" width="29" height="33.4"/><path stroke="#000000" d="M4 4.5h7m3 0h1m3 0h7M4 5.5h1m5 0h1m2 0h1m4 0h1m5 0h1M4 6.5h1m1 0h3m1 0h1m1 0h1m2 0h2m1 0h1m1 0h3m1 0h1M4 7.5h1m1 0h3m1 0h1m1 0h3m3 0h1m1 0h3m1 0h1M4 8.5h1m1 0h3m1 0h1m2 0h1m1 0h1m2 0h1m1 0h3m1 0h1M4 9.5h1m5 0h1m2 0h1m1 0h2m1 0h1m5 0h1M4 10.5h7m1 0h1m1 0h1m1 0h1m1 0h7M14 11.5h1m1 0h1M7 12.5h2m1 0h2m1 0h1m1 0h2m4 0h2M4 13.5h2m1 0h1m3 0h1m1 0h3m2 0h4m2 0h1M5 14.5h4m1 0h1m2 0h2m1 0h1m2 0h1m1 0h1m2 0h1M4 15.5h1m1 0h1m1 0h2m2 0h1m2 0h1m3 0h2M5 16.5h1m1 0h6m3 0h1m2 0h5M12 17.5h4m2 0h1m3 0h1M4 18.5h7m1 0h2m2 0h1m1 0h1m2 0h2M4 19.5h1m5 0h1m11 0h3M4 20.5h1m1 0h3m1 0h1m1 0h1m3 0h1m3 0h1m1 0h1m1 0h1M4 21.5h1m1 0h3m1 0h1m1 0h1m1 0h1m4 0h2M4 22.5h1m1 0h3m1 0h1m5 0h6m1 0h2M4 23.5h1m5 0h1m2 0h1m2 0h4m1 0h1m1 0h2M4 24.5h7m4 0h1m1 0h4m1 0h1"/><text y="30.3" x="50%" text-anchor="middle" font-family="Verdana, 'Bitstream Vera Sans', 'DejaVu Sans', Tahoma, Geneva, Arial, Sans-serif" font-size="4">Caption</text></svg>