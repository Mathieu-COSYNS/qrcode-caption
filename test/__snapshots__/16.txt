data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='500' height='575.8620689655172' viewBox='0 0 29 33.4' shape-rendering='crispEdges'%3e%3crect fill='red' width='29' height='33.4'/%3e%3cpath stroke='black' d='M4 4.5h7m3 0h1m3 0h7M4 5.5h1m5 0h1m2 0h1m4 0h1m5 0h1M4 6.5h1m1 0h3m1 0h1m1 0h1m2 0h2m1 0h1m1 0h3m1 0h1M4 7.5h1m1 0h3m1 0h1m1 0h3m3 0h1m1 0h3m1 0h1M4 8.5h1m1 0h3m1 0h1m2 0h1m1 0h1m2 0h1m1 0h3m1 0h1M4 9.5h1m5 0h1m2 0h1m1 0h2m1 0h1m5 0h1M4 10.5h7m1 0h1m1 0h1m1 0h1m1 0h7M14 11.5h1m1 0h1M7 12.5h2m1 0h2m1 0h1m1 0h2m4 0h2M4 13.5h2m1 0h1m3 0h1m1 0h3m2 0h4m2 0h1M5 14.5h4m1 0h1m2 0h2m1 0h1m2 0h1m1 0h1m2 0h1M4 15.5h1m1 0h1m1 0h2m2 0h1m2 0h1m3 0h2M5 16.5h1m1 0h6m3 0h1m2 0h5M12 17.5h4m2 0h1m3 0h1M4 18.5h7m1 0h2m2 0h1m1 0h1m2 0h2M4 19.5h1m5 0h1m11 0h3M4 20.5h1m1 0h3m1 0h1m1 0h1m3 0h1m3 0h1m1 0h1m1 0h1M4 21.5h1m1 0h3m1 0h1m1 0h1m1 0h1m4 0h2M4 22.5h1m1 0h3m1 0h1m5 0h6m1 0h2M4 23.5h1m5 0h1m2 0h1m2 0h4m1 0h1m1 0h2M4 24.5h7m4 0h1m1 0h4m1 0h1'/%3e%3ctext y='30.3' x='50%25' text-anchor='middle' font-family=%22Verdana%2c 'Bitstream Vera Sans'%2c 'DejaVu Sans'%2c Tahoma%2c Geneva%2c Arial%2c Sans-serif%22 font-size='4' fill='black'%3eCaption%3c/text%3e%3c/svg%3e